.github/stale.yml

```
@@ -0,0 +1,64 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
```

```suggestion
daysUntilClose: 30
```

7 days is too short. The stale PR can easily be missed for a week if someone is on vacation.

---

I think we decided in the meeting that we wouldn't auto close any issues or PRs actually. At least initially.
```suggestion
daysUntilClose: false
```

---

I don't have any recollection of that. I don't see any reason for this tool without autoclosing.